pykokkos/lib/create.py

```diff
@@ -2,3 +2,15 @@
 
 def zeros(shape, *, dtype=None, device=None):
     return pk.View([*shape], dtype=dtype)
+
+
+def ones(shape, *, dtype=None, device=None):
+    if dtype is None:
+        # NumPy also defaults to a double for ones()
+        dtype = pk.float64
+    view: pk.View = pk.View([*shape], dtype=dtype)
+    view[:] = 1
+    if shape == (0,):
+        view.shape = (0,)
+    view.shape = tuple(view.shape)
```

Comment:
I'm a bit confused by what you're trying to do with `view.shape` here. In `_init_view()` we currently do the following:
```Python
        self.shape: Tuple[int] = tuple(shape)
        if self.shape == (0,):
            self.shape = ()
```
Did you make these changes before we changed the shape to a tuple? If so, are they still needed? I'm also not sure why we're forcing `view.shape` to be `(0,)` in the if block